### PR TITLE
Add e2e tests for backendconfig access logging

### PR DESCRIPTION
**Pull request by skmatti**

### CODE REVIEW

File: cmd/e2e-test/logging_test.go [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+	enabled bool
+	// log sampling rate, takes a value between 0.0 and 1.0.
+	sampleRate float64
+}
+
+func TestLogging(t *testing.T) {
+	t.Parallel()
+	const (
+		svcName = "service1"
+	)
```
Comment (rramkumar1) [inline cmd/e2e-test/logging_test.go:49]: Get rid of the parentheses since you only have 1 variable?

Comment (skmatti) [reply cmd/e2e-test/logging_test.go]: Done

**Pull request merged**